src/main/java/teammates/ui/controller/StudentProfileEditSaveAction.java

```diff
@@ -52,7 +52,7 @@ private StudentProfileAttributes extractProfileData() {
         if ("".equals(editedProfile.nationality)) {
             editedProfile.nationality = getRequestParamValue("existingNationality");
         }
-        editedProfile.gender = getRequestParamValue(Const.ParamsNames.STUDENT_GENDER);
+        // editedProfile.gender = getRequestParamValue(Const.ParamsNames.STUDENT_GENDER);
```

Comment:
You probably should use GenderType.valueOf here to convert the variable from string to enum and wrap it inside try-catch block to handle invalid input.